docs/windows/policy/en-US/tailscale.adml

```diff
@@ -42,6 +43,20 @@ To require logging in to a particular tailnet, add the "required:" prefix, such
 If you configure this policy, set it to the name of the tailnet, possibly with the "required:" prefix, as described above.
 
 If you disable this policy, the standard login page will be used.]]></string>
+            <string id="AuthKey">Specify the auth key to authenticate devices without user interaction</string>
+            <string id="AuthKey_Help"><![CDATA[This policy allows specifying the default auth key to be used when registering new devices without requiring sign-in via a web browser, unless the user specifies a different auth key via the CLI.
+
+Managing authentication keys via Group Policy and MDM solutions poses significant security risks. Group Policy is not designed to store and deploy secrets, and by default, Group Policy settings can be read by all domain-authenticated users and devices, regardless of their privilege level or whether the policy setting applies to them.
+
+While MDM solutions tend to offer better control over who can access the policy setting values, they can still be compromised. Additionally, with both Group Policy and MDM solutions, the auth key is always readable by all users who have access to the device where this policy setting applies, as well as by all applications running on the device. A compromised auth key can potentially be used by a malicious actor to gain or elevate access to the target network.
+
+Only consider this option after carefully reviewing the organization's security posture. For example, ensure you configure the auth keys specifically for the tag of the device and that access control policies only grant necessary access between the tailnet and the tagged device. Additionally, consider using short-lived auth keys, Device Approval, and/or Tailnet lock to minimize risk. If you suspect an auth key has been compromised, revoke the auth key immediately.
```

Comment:
Add "one time auth keys" to the list?

Reply:
Added "one-time auth keys (with one GPO/MDM configuration per device)" to clarify how one-time auth keys can be used with GPO/MDM.